src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1010,8 +1010,19 @@ private List<AssociativeNode> GetModifiedNodes(Subtree subtree, bool redefinitio
                     // It can then be handled normally regardless of its ForceExecution state
                     subtree.ForceExecution = false;
 
+                    BinaryExpressionNode prevBNE = null;
+                    if (node is BinaryExpressionNode bne1){
+                        var prevNode = st.AstNodes.OfType<BinaryExpressionNode>().Where(x => (x.LeftNode as IdentifierNode).Name == bne1.LeftNode.Name);
+                        if(prevNode.Count() > 1)
+                        {
+                            throw new Exception("1 prevnode expected");
+                        }
+                        prevBNE = prevNode.FirstOrDefault();
+                    }
```

Comment:
I think it's better to do this search here rather than outside the `if(!nodeFound)`, which was what I was doing.